data/human/derelicts.txt

```diff
@@ -0,0 +1,1793 @@
+				depart
+			label object
+			`	Before you can make it to the airlock something comes flying straight towards you.`
+			choice
+				`	(Try to block.)`
+					goto block
+				`	(Karate chop it.)`
+					goto ninja
+				`	(Close your eyes.)`
+					goto dead
+			label dead
+			`	The object strikes you, sending you sprawling as the ship explodes around you, resulting in your doom.`
+				die
+			label block
+			branch dead
+				random < 90
+			`	You just manage to block the object, allowing you just enough time to make it to the airlock. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart	
+			label ninja
+			branch dead
+				random < 90
+			`	By some miracle your karate chop deflects the object, allowing you just enough time to make it to the airlock. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label success
+			action
+				payment 65000
+			`	Despite your best efforts to make the ship actually fly it just won't, but now you have all the time to loot as you please and even pilfer some outfits from the ship. However, before leaving the cockpit you find a credit chip sitting in the cockpit worth <payment>.`
+				decline			
+			label salvage
+			`	Deciding not to risk it you quickly scour the ship for valuable scrap and outfits.`
+			branch valuable
+				random < 35
+			action
+				payment 22500
+			`	Upon a brief inspection of the ship you find little valuable, only managing to salvage a meager <payment> credits from the <npc>. Before you depart you detonate the ship to prevent any other derelict hunters from scouring an empty ship.`
+				depart
+			label valuable
+			branch rare
+				random < 5
+			action
+				payment 63000
+			`	Searching through the ship you find that it is quite valuable and you manage to salvage a substantital amount from the <npc>, worth <payment> credits. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
+				depart
+			label rare
+			branch android
+				random < 85
+			action
+				payment 475000
+			`	Hidden away under a panel you find a small box. Opening it reveals a huge amount of credits worth <payment> credits! Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
+				depart
+			label android
+			action
+				outfit "NDR-114 Android" 1
+				payment 22500
+			`	As you search the ship for anything valuable you find some kind of robot tucked into a room, you turn it on, commanding it to head to your ship allowing you time to further salvage a meager <payment> credits from the <npc>. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
+				depart
+		fleet
```

Comment:
I don't know if it makes a difference, but the `fleet` should go before the conversation, as elsewhere.